.github/workflows/ci.yml

```diff
@@ -24,7 +24,7 @@ jobs:
         eslint:
           - '8.56'
           - '8'
-          - '9'
```

Comment:
[Yesterday the task was working](https://github.com/un-ts/eslint-plugin-import-x/actions/runs/11843157078/job/33003459670)

[Today is not](https://github.com/un-ts/eslint-plugin-import-x/actions/runs/11862291324)

It seems an issue with `eslint`: https://github.com/eslint/eslint/issues/19134.

I assume that the following steps in CI workflows ends up installing the  last version of `eslint`:

https://github.com/un-ts/eslint-plugin-import-x/blob/14fc608ddc53dd313a90320538714eee692e8be3/.github/workflows/ci.yml#L56-L63

https://github.com/un-ts/eslint-plugin-import-x/blob/14fc608ddc53dd313a90320538714eee692e8be3/.github/workflows/ci.yml#L24-L27

Replacing `9` with `9.14` makes CI green again.

---

I just want to report that on the [failing CI](https://github.com/un-ts/eslint-plugin-import-x/actions/runs/11862291324/job/33061374177) the `eslint` version printed in the error is `9.8.0` which seems wrong since the error is present on `9.15.0`.

Reply:
It seems that the issue comes from the `typescript-eslint` now. Let's wait and see if bumping `typescript-eslint` is possible.

Reply:
Yeah, and the same issue is affecting `eslint-plugin-unicorn`: https://github.com/sindresorhus/eslint-plugin-unicorn/issues/2496

Reply:
[Last CI is green with eslint matrix `9`](https://github.com/un-ts/eslint-plugin-import-x/actions/runs/11943879122) marking as resolved!